source/clear-linux/reference/collaboration/community/mailing.rst

```diff
@@ -0,0 +1,75 @@
+.. _mailing:
+
+Mailing list guidelines
+=======================
+
+These guidelines apply for all emails written to dev@lists.clearlinux.org.
+They ensure communication remains civil, productive, and brief. Use the
+`mailing list`_ effectively and get as much feedback as possible on your
+work.
+
+* Use the mailing list as much as possible. Specifically, use the mailing
+  list to:
+
+   + Submit :abbr:`RFCs(Requests for Comments)` on planned contributions.
+   + Submit bugs for the team to address.
```

Comment:
Lets have this link to the bug report template

Reply:
agree that we should reference back to the bug report template

Reply:
done